src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/CoreCLRReadyToRunEntryPointTableNode.cs

```diff
@@ -0,0 +1,203 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Internal.NativeFormat;
+using Internal.Runtime;
+using Internal.Text;
+using Internal.TypeSystem;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    public class CoreCLRReadyToRunEntryPointTableNode : ObjectNode, ISymbolDefinitionNode
```

Comment:
Nit: Prefixing every other type name with `CoreCLRReadyToRun` makes the names very long.

Reply:
Makes sense. I'll talk to @nattress, I think that something like "R2REntryPointTableNode" would suffice unless you have better suggestions.

Reply:
Would it make sense to use namespace?